docker-entrypoint.sh

```diff
@@ -32,17 +32,30 @@ if [ "${1}" = 'dev' -o "${1}" = "production" -o "${1:0:2}" = "--" ]; then
 fi
 
 CONFIG_FILE=/usr/lib/supertokens/config.yaml
+TEMP_LOCATION_WHEN_READONLY=/lib/supertokens/temp/
 CONFIG_MD5SUM="$(md5sum /usr/lib/supertokens/config.yaml | awk '{ print $1 }')"
 
-# if files have been shared using shared volumes, make sure the ownership of the
-# /usr/lib/supertokens files still remains with supertokens user
-chown -R supertokens:supertokens /usr/lib/supertokens/
+#if it contains the readonly param, make that temp dir a temp dir for java io too
+if [[ "$*" == *--read-only* ]]
```

Comment:
Can't we always assume read only. I mean, the user just needs to mount the temp dir if it's a read only fs. otherwise doesn't.